### DISCUSSION TITLE
Meeting notes 19/06

### DISCUSSION THREAD

**Discussion by Ivanderkelen** [CORDEX-be2/ValEnsPy#75] [Meeting minutes]
## Agenda

0. Tour de table - usage and needs?
1. Next development focus points? 
2. CLIMATE_GRID input generator: update @Ivanderkelen 
3. Preprocessor: regridding? 
4. InputManager: Update
- Data storing structures - mix of mean, min, max (ERA-5 Daily data) #69 Notebook

5. Merging Settings -> Move to [squash merging](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/about-pull-request-merges#squash-and-merge-your-commits) 

6. Time-axis constraints: pandas datetime or beyond? #63 

## Meeting notes
@wdewettin, @kbruyndonckx @bertvs1 @kobebryant432 and @Ivanderkelen  joined the meeting

- Wout will start with the verification of his 30 year runs. (4x different resolution). and will use Valenspy, especially CLIMATE_GRID input convertor. He is keen on adding RMSE and other metrics to the source code. (esp on his analysis using IDF curves, PoT and annual maxima)
- Karian is doing water cycle evaluation of CORDEX BE 1 data. 

- Preprocessor: regridding step will happen in there (in addition to masking, extracting point data and other preprocessing steps). The input convertors will provide the output on the native grids (eg rotated latitude and longitude for RCM simulations). Important that the information on their grids is passed in arguments. to do for @kobebryant432 

- Discussion on time-axis (datetime64, numpy type). Decision: keep flexible to read in different calendars, BUT datetime64 is standard (CF checker should give warning if not datetime64). 

- Idea to make overview table on observational datasets: new issue #82

- Discussion on ERA5 structure, see #62 

### Priorities
- working on the preprocessor and getting the first users up and running!

**Comment by kobebryant432** [comment]
Maybe include Wout, Fien & Karian for this meeting? As they are main users it may be useful to see where they are at, what they are waiting for and how we could integrate them more into the process. @bertvs1 @Ivanderkelen

**Reply by Ivanderkelen** [reply]
Good idea, I'll forward them the invite.
